model/trace/instrumentation/graphql.yml

```diff
@@ -24,9 +24,63 @@ groups:
             - id: subscription
               value: "subscription"
               brief: "GraphQL subscription"
-        examples: ['query', 'mutation', 'subscription']
-      - id: document
-        brief: "The GraphQL document being executed."
+        examples: ["query", "mutation", "subscription"]
+      - id: document.content
+        brief: "The content of the GraphQL document being executed."
         type: string
         note: The value may be sanitized to exclude sensitive information.
-        examples: 'query findBookById { bookById(id: ?) { name } }'
+        examples: "query findBookById { bookById(id: ?) { name } }"
+      - id: document.sha256
+        brief: "The SHA256 hash of the document.content"
+        type: string
+        examples: "400483f38c08e8a3d3b972409c9dfb8e4a326e1b1940864932acd9f873d8664c"
+      - id: document.id
+        brief: >
+          The id of the document beeing executed. This is a hash of the document provided by the user to identitfy 
+          persisted queries.
+        type: string
+        examples: "aa3e37c1bf54708e93f12c137afba004"
+      - id: error.count
+        brief: "The number of errors that occurred during the operation."
+        type: int
+        examples: 3
+    # TODO should we have something like outcome (success, failure)
+    # TODO how do we specify errors?
```

Comment:
A GraphQL Operation can have multiple errors in the response:
```json
{
  "data": null,
  "errors": [
    {
      "message": "Cannot return null for non-nullable field User",
      "locations": [
        {
          "line": 3,
          "column": 5
        }
      ],
      "path": [
        "createUser",
        "user"
      ],
      "extensions": {
        "code": "USERNAME_INVALID",
        "message": "username is invalid"
      }
    }
  ]
}
```

What is the best way to represent this in OpenTelemetry?

Reply:
I'd say maybe use the attributes under the `exception` namespace: https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/exception.md